doc/source/user_guide/options.rst

```diff
@@ -487,6 +487,8 @@ styler.sparse.index                     True         "Sparsify" MultiIndex displ
                                                      elements in outer levels within groups).
 styler.sparse.columns                   True         "Sparsify" MultiIndex display for columns
                                                      in Styler output.
+styler.max.elements                     262144       Maximum number of datapoints that Styler will render
```

Comment:
make the default much lower like 10000

Reply:
actually is there a reason we don't have row and column limits like other display? eg we don't do element limits anywhere else

Reply:
There was previous discussion at the end of #11700. The conclusion was:

- Styler is typically displayed in browser (jupyter) so it can display much more effectively than a console. (262144 = 2**18) was chosen as the maximal value subject to a total rendering time constraint, currently around 8s. 10000 elements renders in 0.2s.

- Using the much smaller `display.max_columns`, `display.max_rows` would significantly reduce the default display capacity of Styler unnecessarily.

On the new option, `max_elements` is used to dynamically derive `max_rows` and `max_columns` for different aspect ratio dataframes that is passed to rendering. We don't have this anywhere else: consider it a technological advancement :) . With the advantages: only one option, only trims when absolutely necessary.